README.md

```diff
@@ -154,7 +154,7 @@ Creates a new record if none exists.
 | Param | Type | Description |
 | --- | --- | --- |
 | data | <code>Object</code> | Data to store in the given DynamoDB table. Each top-level property will   become a top-level attribute in the DynamoDB table and will replace any   existing top-level attribute with the same name entirely. We'd like to   allow partial attribute updates which would recursively replace the   structure provided while leaving any missing sub-properties untouched,   but unfortunately we are prevented by this issue:   https://forums.aws.amazon.com/thread.jspa?threadID=162907 |
-| primaryKey | <code>Array</code> | An object containing partitionKey and sortKey key/value properties. |
+| primaryKey | <code>Object</code> | An object containing partitionKey and sortKey key/value properties.   DynamoDB also sometimes calles the partition key the "HASH" key and the   sort key the "RANGE" key. For example, if your DynamoDB table/index had   a partition/hash key of "user_id" and a sort/range key of "created", then   your primaryKey parameter would look something like this:   { user_id: '123', created: 1493741395330 } |
```

Comment:
I think there is a way to express examples in jsdoc that we could use here: http://usejsdoc.org/tags-example.html